plugins/hypervisors/xenserver/test/com/cloud/hypervisor/xenserver/resource/CitrixHelperTest.java

```diff
@@ -0,0 +1,35 @@
+package com.cloud.hypervisor.xenserver.resource;
+
+import junit.framework.Assert;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```

Comment:
import * is not accepted per the style guidelines enforced by checkstyle. please replace.
```
[INFO] Starting audit...
/Users/daan/cloudstack/cloudstack/plugins/hypervisors/xenserver/test/com/cloud/hypervisor/xenserver/resource/CitrixHelperTest.java:6: Using the '.*' form of import should be avoided - org.junit.Assert.*.
Audit done.
```
hence
```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 07:29 min
[INFO] Finished at: 2015-12-12T14:18:54+01:00
[INFO] Final Memory: 93M/788M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:2.11:check (cloudstack-checkstyle) on project cloud-plugin-hypervisor-xenserver: Failed during checkstyle execution: There are 1 checkstyle errors. -> [Help 1]
```